ios/FluentUI/Button/ButtonTokenSet.swift

```diff
@@ -82,29 +82,29 @@ public class ButtonTokenSet: ControlTokenSet<ButtonTokenSet.Tokens> {
                 return .dynamicColor {
                     switch style() {
                     case .accent:
-                        return theme.aliasTokens.colors[.brandBackground1]
```

Comment:
I assume you're using Button as the example control to demonstrate the new APIs, and in the future you'll be transitioning all controls to the new APIs?

Reply:
Yep, that's the idea! I wanted to keep the PR digestible (which a massive find+replace definitely will NOT be), but also prove that it works via a moderately complex component.